src/error.rs

```diff
@@ -74,8 +65,17 @@ impl std::error::Error for Error {
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        use std::error::Error;
-        write!(f, "Hammersbald error: {} cause: {:?}", self.description(), self.source())
+        // use std::error::Error;
```

Comment:
I think you forgot to remove this line :)

Reply:
Thanks for spotting that one.